src/js/player.js

```diff
@@ -274,7 +263,54 @@ class Player extends Component {
 
     // Initialize tech instance
     let techComponent = Component.getComponent(techName);
-    this.tech = new techComponent(this, techOptions);
+    this.tech = new techComponent(techOptions);
+
+    this.on(this.tech, 'ready', this.onTechReady);
+    this.on(this.tech, 'usenativecontrols', this.onTechUseNativeControls);
+
+    // firefox doesn't bubble mousemove events to parent. videojs/video-js-swf#37
+    // bugzilla bug: https://bugzilla.mozilla.org/show_bug.cgi?id=836786
```

Comment:
The firefox bug there says it's resolved, so we should test if we can remove this.